src/SWP/Bundle/BridgeBundle/Resources/config/doctrine-orm/Group.orm.yml

```diff
@@ -0,0 +1,32 @@
+SWP\Component\Bridge\Model\Group:
+    type: mappedSuperclass
+    table: swp_item_groups
+    repositoryClass: SWP\Bundle\StorageBundle\Doctrine\ORM\EntityRepository
+    gedmo:
+        soft_deleteable:
+            field_name: deletedAt
+            time_aware: false
+    id:
+        id:
+            type: integer
+            id: true
+            generator:
+                strategy: AUTO
+    fields:
+        code:
+            type: string
+        deletedAt:
+            type: datetime
+            nullable: true
+    oneToMany:
+        items:
```

Comment:
We will have issues with it. This way it's always lazy loaded when group is loaded (if not already joined in query). We have problem like that with content list items. Imo we should remove it and work with items through service.